### DISCUSSION TITLE
Proper repo for customization?

### DISCUSSION THREAD

**Discussion by wespiard** [manna-harbour/miryoku#103] [Q&A] [answered]
I've been reading around for old and new methods of customizing layers/keys. I somewhat understand the process based on #85, but I'm not sure which repo/branch I should be using to get the most up-to-date results. 

QMK master seems to not have these custom files or the babel folder. Should I just switch to the `miryoku` branch of the main QMK repo, or perhaps fork my own copy of the Miryoku QMK repo?

Basically, I'd like to add a custom gaming layer (not just the QWERTY tap-only layer, but similar), change some of the thumb buttons around (I like space on the right side, and I don't need the mouse/button layers).

**Comment by manna-harbour** [comment, marked as answer]
Currently for customisation of Miryoku QMK using #85, you need to be using the `miryoku` branch. I've just added a note to the docs. Also I'll be making a PR soon, and after that you'll be able to use the feature from `master`. 

In general, to use the most recently added features you'll need to use `miryoku`, but I'll try to make PRs frequently so `master` won't get too far behind.

If you also need recent features from `master` that aren't in `miryoku`, such as new keyboards, you can merge both branches. See the Miryoku QMK workflow docs for an easy way to do that. Also I'll be rebasing `miryoku` on `master` soon. 

To build locally it's probably best to add a remote, but to use the workflows you'll need to fork.
